### PR TITLE
prune old status numbers

### PR DESCRIPTION
**Pull request by staltz**
When emitting on the `status` obv, check which stats were updated "long ago" and omit them. For issue #102

### CODE REVIEW

File: status.js [hunk 1]
```diff
@@ -2,12 +2,24 @@ const Obv = require('obz')
 
 module.exports = function Status() {
   const indexesStatus = {}
+  const indexesLastTime = {}
```
Comment (arj03) [inline status.js:5]: You said in an earlier PR that maps would be better when deleting. Should we do that here as well? Otherwise this looks good.

Comment (staltz) [reply status.js:5]: Yeah I was thinking about that. If we use a Map, we have to convert it to an object every time we emit the obv. I thought that maybe that's wasteful (object allocation for every obv emit). Gladly the obv emit doesn't happen frequently, so here it was a choice RAM or CPU, and I don't think the CPU will be significantly hurt.

Comment (arj03) [reply status.js:5]: Alright, lets leave it like this then. Don't think it will be a problem in practise :)

**Review by arj03 — APPROVED** [review]

File: status.js (hunk 1 above)
Comment (staltz) [reply status.js:5]: Hmm, maybe it's fine if the obv emits Maps, because this is not crossing any muxrpc boundary

Comment (staltz) [reply status.js:5]: Yeah, bigger fish to catch, but this is up for discussion in the future.

Comment (arj03) [reply status.js:5]: It's up to you, I'm fine either way.

**Pull request merged**